### DISCUSSION TITLE
Welcome to island.js Discussions!

### DISCUSSION THREAD

**Discussion by sanyuan0704** [sanyuan0704/island.js#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baiwusanyu-c** [comment]
簡單看了下這個是不是可以大致理解為 react 版本的 vitepress ？

**Reply by sanyuan0704** [reply]
可以这么理解，细节层面会有很多差异，比如插件机制、搜索、国际化
